### PR TITLE
Failed to export classes for TextClasDataBunch.from_ids

### PR DESCRIPTION
**Pull request by troilusc**
TextClasDataBunch generated by `TextClasDataBunch.from_ids` has not set up the CategoryProcessor, so the info of classes get lost when export and load it again.

Let me know what needs to be adjusted or changed (My first pull request, so I assume there will a lot.).

### CODE REVIEW

File: tests/test_text_data.py [hunk 1]
```diff
@@ -186,6 +186,16 @@ def test_from_ids_works_for_variable_length_sentences():
                                       train_ids=ids, train_lbls=lbl,
                                       valid_ids=ids, valid_lbls=lbl, classes={0:0}, bs=8)
 
+def test_from_ids_exports_classes():
+    this_tests(TextClasDataBunch.from_ids)
+    ids = [np.array([0])]*10
+    lbl = [0]*10
+    data = TextClasDataBunch.from_ids('/tmp', vocab=Vocab({0: BOS, 1:PAD}),
+                                      train_ids=ids, train_lbls=lbl,
+                                      valid_ids=ids, valid_lbls=lbl, classes={0:0}, bs=8)
```
Comment (sgugger) [inline tests/test_text_data.py]: Here classes should be a list, like ['a', 'b', 'c']

File: tests/test_text_data.py [hunk 2]
```diff
@@ -186,6 +186,16 @@ def test_from_ids_works_for_variable_length_sentences():
...
+    data.export('/tmp/export.pkl')
+    TextClasDataBunch.load_empty('/tmp').classes
```
Comment (sgugger) [inline tests/test_text_data.py]: Make an assert to test we get back the same thing as we passed before.

**Comment by troilusc** [comment]
Thanks for your suggestion! I've updated the test accordingly.

**Comment by sgugger** [comment]
Thanks a lot!

**Comment by troilusc** [comment]
My pleasure!

**Pull request merged**